tests/ctst/steps/azureArchive.ts

```diff
@@ -4,9 +4,9 @@ import assert from 'assert';
 import { safeJsonParse } from '../common/utils';
 import { Given, Then, When, After, setDefaultTimeout } from '@cucumber/cucumber';
 import { AzureHelper, S3, Constants, Utils } from 'cli-testing';
+import { exec } from 'child_process';
```

Comment:
Do you think a work should be done on CTST to make it simpler to run other cli tools like sorbetclt here? With a set of methods, from CTST, to execute a tool with command line parameters and then handle the errors for you?

Reply:
Yep, as we discussed in the office on friday, I would say that an implementation of kube api would be very nice here